src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandler.java

```diff
@@ -42,19 +42,20 @@
     public NewHandler(IndentationCheck indentCheck,
                       DetailAST ast,
                       AbstractExpressionHandler parent) {
-        super(indentCheck, "operator new", ast, parent);
+        super(indentCheck, "new", ast, parent);
         mainAst = ast;
     }
 
     @Override
     public void checkIndentation() {
         // if new is on the line start and it is not the part of assignment.
-        if (isOnStartOfLine(mainAst)
-                && !isNewKeywordFollowedByAssign()) {
+        if (isOnStartOfLine(mainAst)) {
             final int columnNo = expandedTabsColumnNo(mainAst);
             final IndentLevel level = getIndentImpl();
 
-            if (columnNo < level.getFirstIndentLevel()) {
+            final boolean isForceStrictCond = getIndentCheck().isForceStrictCondition();
```

Comment:
`isForceStrictCond` is a bad name for a variable. Why not `isForceStrictCondition` or `forceStrictCondition`?
If a line does not fit, you can add overload to `isAcceptable` method. Something like `IndentLevel.isAcceptable(int indent, boolean isForceStrictCondition)`. Then it will add a bit of readability.

Reply:
Done.

Reply:
Please undo new method